docs/source/tutorials/token_classification/rank.py

```diff
@@ -0,0 +1,103 @@
+import pandas as pd 
+import numpy as np 
+from cleanlab.rank import get_label_quality_scores as main_get_label_quality_scores 
+
+def get_label_quality_scores(labels: list, 
+                             pred_probs: list, 
+                             method: str = "min", 
+                             cleanlab_method: str = "self_confidence", 
+                             param: float = 0.04, 
+                             tokens: list = None, 
+                             return_token_info: bool = False): 
```

Comment:
Applies to all the functions: To ensure forwards compatibility of users' code if we later decide to reorder some args, we should make all optional args keyword-only args rather than allowing them to be positionally specified. For this function in particular (you should also change all other functions similarly, this means you should change the args to look like this:

```
def get_label_quality_scores(labels: list, pred_probs: list, *,  
                             method: str = "min", 
                             cleanlab_method: str = "self_confidence", 
                             param: float = 0.04, 
                             tokens: list = None, 
                             return_token_info: bool = False):
```